src/Javis.jl

```diff
@@ -2,6 +2,8 @@ module Javis
 
 using Luxor, LaTeXStrings
 
+include("backgrounds.jl")
```

Comment:
this should probably be done after the type definitions as you have `::Video` in there for example which is unknown at that point. Probably want to move t to the point where I have the other `include`

Reply:
Ah. That fixed an error I was encountering. Everything is good now.